zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -362,6 +362,18 @@ pub async fn test_responses<State, ReadState>(
         .expect("unexpected error in submitblock RPC call");
 
     snapshot_rpc_submit_block_invalid(submit_block, &settings);
+
+    // `validateaddress`
+    let founder_address = match network {
+        Network::Mainnet => "t3fqvkzrrNaMcamkQMwAyHRjfDdM2xQvDTR",
+        Network::Testnet => "t2UNzUUx8mWBCRYPRezvA363EYXyEpHokyi",
+    };
+
+    let validate_address = get_block_template_rpc
+        .validate_address(founder_address.to_string())
+        .await
+        .expect("We should have a validate_address::Response");
+    snapshot_rpc_validateaddress(validate_address, &settings);
```

Comment:
Can you add a snapshot test for an invalid address as well?

This makes sure that all the fields are skipped when they should be.

Reply:
Good idea. Added in [`08ce203`](https://github.com/ZcashFoundation/zebra/pull/6086/commits/08ce20354dd0d9a51c4281c91b72c4617a362f4d).